docker-compose-https.yml

```diff
@@ -2,6 +2,9 @@ version: '3'
 services:
   weblate:
     image: weblate/weblate
+    # To use a custom port, change the first 8080 to your port
+    ports:
+    - 8080:8080
```

Comment:
There is no need to expose port here, it is done via `https-portal` below.

Reply:
> There is no need to expose port here, it is done via `https-portal` below.

Yes, I could not access Weblate until I published the ports